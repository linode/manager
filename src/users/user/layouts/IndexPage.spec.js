describe('components/IndexPage', () => {
  it.skip('should render without errro', () => {
  });
});
