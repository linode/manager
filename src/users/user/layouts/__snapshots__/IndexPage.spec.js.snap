// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/IndexPage should render without errro 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <IndexPage
    dispatch={[Function]}
    user={
      Object {
        "_polling": false,
        "email": "example1@domain.com",
        "password": "crisprcas9",
        "restricted": false,
        "username": "testuser1",
      }
    }
  >
    <div />
  </IndexPage>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <SideEffect(ChainedDocumentTitle)
          title="testuser1"
        />,
        <header
          className="main-header"
        >
          <div
            className="container"
          >
            <div
              className="float-sm-left"
            >
              <Link
                replace={false}
                to="/users"
              >
                Users
              </Link>
              <h1
                title="testuser1"
              >
                <Link
                  replace={false}
                  to="/users/testuser1"
                >
                  testuser1
                </Link>
              </h1>
            </div>
          </div>
        </header>,
        <div
          className="main-header-fix"
        />,
        <Tabs
          className=""
          isSubTabs={false}
          onClick={[Function]}
          pathname="blank"
          tabs={
            Array [
              Object {
                "link": "/users/testuser1",
                "name": "Dashboard",
              },
            ]
          }
        >
          <div />
        </Tabs>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "title": "testuser1",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="container"
          >
            <div
              className="float-sm-left"
            >
              <Link
                replace={false}
                to="/users"
              >
                Users
              </Link>
              <h1
                title="testuser1"
              >
                <Link
                  replace={false}
                  to="/users/testuser1"
                >
                  testuser1
                </Link>
              </h1>
            </div>
          </div>,
          "className": "main-header",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="float-sm-left"
            >
              <Link
                replace={false}
                to="/users"
              >
                Users
              </Link>
              <h1
                title="testuser1"
              >
                <Link
                  replace={false}
                  to="/users/testuser1"
                >
                  testuser1
                </Link>
              </h1>
            </div>,
            "className": "container",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <Link
                  replace={false}
                  to="/users"
                >
                  Users
                </Link>,
                <h1
                  title="testuser1"
                >
                  <Link
                    replace={false}
                    to="/users/testuser1"
                  >
                    testuser1
                  </Link>
                </h1>,
              ],
              "className": "float-sm-left",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "Users",
                  "replace": false,
                  "to": "/users",
                },
                "ref": null,
                "rendered": "Users",
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <Link
                    replace={false}
                    to="/users/testuser1"
                  >
                    testuser1
                  </Link>,
                  "title": "testuser1",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "testuser1",
                    "replace": false,
                    "to": "/users/testuser1",
                  },
                  "ref": null,
                  "rendered": "testuser1",
                  "type": [Function],
                },
                "type": "h1",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "main-header-fix",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <div />,
          "className": "",
          "isSubTabs": false,
          "onClick": [Function],
          "pathname": "blank",
          "tabs": Array [
            Object {
              "link": "/users/testuser1",
              "name": "Dashboard",
            },
          ],
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <SideEffect(ChainedDocumentTitle)
            title="testuser1"
          />,
          <header
            className="main-header"
          >
            <div
              className="container"
            >
              <div
                className="float-sm-left"
              >
                <Link
                  replace={false}
                  to="/users"
                >
                  Users
                </Link>
                <h1
                  title="testuser1"
                >
                  <Link
                    replace={false}
                    to="/users/testuser1"
                  >
                    testuser1
                  </Link>
                </h1>
              </div>
            </div>
          </header>,
          <div
            className="main-header-fix"
          />,
          <Tabs
            className=""
            isSubTabs={false}
            onClick={[Function]}
            pathname="blank"
            tabs={
              Array [
                Object {
                  "link": "/users/testuser1",
                  "name": "Dashboard",
                },
              ]
            }
          >
            <div />
          </Tabs>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "title": "testuser1",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="container"
            >
              <div
                className="float-sm-left"
              >
                <Link
                  replace={false}
                  to="/users"
                >
                  Users
                </Link>
                <h1
                  title="testuser1"
                >
                  <Link
                    replace={false}
                    to="/users/testuser1"
                  >
                    testuser1
                  </Link>
                </h1>
              </div>
            </div>,
            "className": "main-header",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <div
                className="float-sm-left"
              >
                <Link
                  replace={false}
                  to="/users"
                >
                  Users
                </Link>
                <h1
                  title="testuser1"
                >
                  <Link
                    replace={false}
                    to="/users/testuser1"
                  >
                    testuser1
                  </Link>
                </h1>
              </div>,
              "className": "container",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <Link
                    replace={false}
                    to="/users"
                  >
                    Users
                  </Link>,
                  <h1
                    title="testuser1"
                  >
                    <Link
                      replace={false}
                      to="/users/testuser1"
                    >
                      testuser1
                    </Link>
                  </h1>,
                ],
                "className": "float-sm-left",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "children": "Users",
                    "replace": false,
                    "to": "/users",
                  },
                  "ref": null,
                  "rendered": "Users",
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <Link
                      replace={false}
                      to="/users/testuser1"
                    >
                      testuser1
                    </Link>,
                    "title": "testuser1",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "children": "testuser1",
                      "replace": false,
                      "to": "/users/testuser1",
                    },
                    "ref": null,
                    "rendered": "testuser1",
                    "type": [Function],
                  },
                  "type": "h1",
                },
              ],
              "type": "div",
            },
            "type": "div",
          },
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "main-header-fix",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <div />,
            "className": "",
            "isSubTabs": false,
            "onClick": [Function],
            "pathname": "blank",
            "tabs": Array [
              Object {
                "link": "/users/testuser1",
                "name": "Dashboard",
              },
            ],
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
